Drop the last test since it has file sorting issues.

--- the_silver_searcher-1.0.1/tests/empty_match.t
+++ the_silver_searcher-1.0.1/tests/empty_match.t
@@ -11,12 +11,3 @@
 A genuine zero-length match should succeed:
   $ ag "^" nonempty.txt
   1:foo
-
-Empty files should be listed with --unrestricted --files-with-matches (-ul)
-  $ ag -lu --stats | sed '$d' # Remove the last line about timing which will differ
-  empty.txt
-  nonempty.txt
-  2 matches
-  2 files contained matches
-  2 files searched
-  4 bytes searched
